doc/source/rayserve/overview.rst

```diff
@@ -17,21 +17,21 @@ RayServe is a scalable model-serving library built on Ray.
 
 For users RayServe is:
 
-- **Framework Agnostic**:Use the same toolkit to serve everything from deep learning models 
+- **Framework Agnostic**:Use the same toolkit to serve everything from deep learning models
```

Comment:
There are 20 lines of just removing trailing whitespace. The added content is down below